frontend/packages/console-shared/src/components/formik-fields/multi-column-field/multicolumn-field-utils.ts

```diff
@@ -0,0 +1,21 @@
+import * as _ from 'lodash';
+import { gridItemSpanValueShape } from '@patternfly/react-core';
+
+export const getSpans = (totalFieldCount: number): gridItemSpanValueShape[] => {
+  const spans: gridItemSpanValueShape[] = _.fill(Array(totalFieldCount), 1);
+  if (totalFieldCount === 1) {
+    return [12];
+  }
+  if (totalFieldCount === 12) {
+    return _.fill(Array(12), 1);
+  }
+  while (_.sum(spans) !== 12) {
+    for (let i = 0; i < totalFieldCount; i++) {
+      spans[i]++;
+      if (_.sum(spans) === 12) {
+        break;
+      }
+    }
+  }
+  return spans;
+};
```

Comment:
This can be simplified to something like this:
```
const spans = _.fill(Array(totalFieldCount), Math.trunc(12 / totalFieldCount));
let remainder = 12 % totalFieldCount;
while (remainder--) {
  spans[remainder]++;
}
return spans;
```